source/_integrations/unifiled.markdown

```diff
@@ -0,0 +1,55 @@
+---
+title: "UniFi LED"
+description: "Instructions on how to configure the UniFi LED integration with UniFi LED Controller by Ubiquiti."
+logo: ubiquiti.png
+ha_category:
+  - Light
+ha_release: 0.101
+ha_iot_class: Local Polling
+---
+
+[UniFi LED](https://unifi-led.ui.com/) by [Ubiquiti Networks, inc.](https://www.ubnt.com/) is a system off controler managed led light panels and dimmers.
+
+There is currently support for the following device type within Home Assistant:
+
+- [Light](#light)
+
+## Configuration
+
+This integration uses settings in the configuration file. Underneath here is a example.
+
+```yaml
+# Example configuration.yaml entry
+light:
+  - platform: unifiled
+    host: IP_ADDRESS
+    username: USERNAME
+    password: PASSWORD
+```
+
+{% configuration %}
+host:
+  description: Ip address or DNS name used to connect to the Unifi LED controller.
+  type: string
+  required: true
+  default: None
+port:
+  description: Port used to connect to the Unifi LED controller.
```

Comment:
Is it necessary to make this configurable at all? Other than that, description such as `Control Port`, `Login username` would be shorter and easier to read.